.github/workflows/playwright-integration-tests-postgres.yml

```diff
@@ -14,17 +14,18 @@
 
 name: Postgres Playwright Integration Tests
 on:
+  pull_request:
```

Comment:
Hey @TeddyCr - Do we need any constraints here on which file changes to run this CI or which paths to ignore ?